lib/msal-common/src/request/CommonRefreshTokenRequest.ts

```diff
@@ -21,4 +21,5 @@ export type CommonRefreshTokenRequest = BaseAuthRequest & {
     refreshToken: string;
     tokenQueryParameters?: StringDict;
     ccsCredential?: CcsCredential;
+    forceCache?: boolean;
```

Comment:
Can we add an explanation of this new parameter above?

Reply:
For sure. Added.